Esri/arcgis-runtime-samples-android
# Android Code Reformatting

## Description
- Ran file-reformat on all the Kotlin and Java samples
- Code cleanup on the entire project
- Updated Kotlin samples to use 4 spaces for indentation
- Removed unused imports, resources, and fixed code style issues.

## Links and Data
https://devtopia.esri.com/runtime/common-samples/issues/3196

## What To Review
Review the changes made to the code using the Android code inspection tools

## How to Test
The sample behavior should not change in this PR, so test if the samples build and run without errors.
